packages/meditrak-app-server/src/__tests__/__integration__/sync/PushChangesRoute.test.ts

```diff
@@ -217,9 +230,25 @@ describe('changes (POST)', () => {
           body: [imageAction],
         });
         expect(imagePostResponse.statusCode).toEqual(200);
+      });
+    });
+
+    describe('Survey responses containing files', () => {
+      it('correctly uploads a file', async () => {
+        const uniqueFileName = `${generateId()}_file.png`;
+        const fileResponseObject = { uniqueFileName, data: TEST_IMAGE_DATA };
 
-        const imageString = mockS3Bucket[id];
-        expect(imageString).toEqual(TEST_IMAGE_DATA);
```

Comment:
The actual implementation of S3Client differs, this mock is making an incorrect assumption that image data is unchanged. In reality, it strips the data uri prefix. I have changed the mock to just record which inputs have been passed to S3Client by id and fileName for images and files respectively.